lib/hammer_cli_csv/products.rb

```diff
@@ -50,99 +59,179 @@ def import
         end
       end
 
-      # FIXME: TODO remove this rubocop
-      # rubocop:disable CyclomaticComplexity
       def create_products_from_csv(line)
         return if option_organization && line[ORGANIZATION] != option_organization
 
+        count(line[COUNT]).times do |number|
```

Comment:
hammer csv started out as a dev tool to allow the export of data from a running server and then turn around and import it into a new one (new vagrant up). Also part of this was an easy way to create massive amounts of test data. To easily do this, a single row can be iterated "Count" column number of times. Some columns will then allow their values to be of the form "host%d.example.com" which would result in "host0.example.com", "host1...", etc. Count column is optional and defaults to one.

'number' is probably a poor choice in hindsight but lump it under tech debt since it is in every file by now. :)